javascript/javascript3/week2/movies.js

```diff
@@ -0,0 +1,14 @@
+const badMoviesList = document.querySelector('#bad_movies');
+
+fetch('https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json')
+    .then(response => response.json())
+    .then(movies => {
+        const badMovies = movies.filter(movie => movie.rating <= 3)
+        const badMoviesSince2000 = badMovies.filter(movie => movie.year >= 2000)
```

Comment:
good job! instead of creating two filter function, you can use and operator to check both conditions in one function.

Reply:
thank you for your feedback. I will make all the changes.